org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/dom/SchemaLocationHint.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2018 Red Hat Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ * Red Hat Inc. - initial API and implementation
+ */
+package org.eclipse.lemminx.dom;
+
+/**
+ * Represents one of the location hints provided in an xsi:schemaLocation attribute.
+ */
+public class SchemaLocationHint implements DOMRange {
+
+	private final int start, end;
+
+	private final String hint;
+
+	private final SchemaLocation parent;
+
+	/**
+	 * Create a new SchemaLocationHint object that represents one of the location hints
+	 * given in the parent <code>SchemaLocation</code>.
+	 *
+	 * @param start The offset from the beginning of the document where this location hint starts
+	 * @param end The offset from the beginning of the document where this location hint ends
+	 * @param hint The hint to the location of a schema (A URI that points to a schema)
+	 * @param parent The <coed>SchemaLocation</code> in which this hint was given
```

Comment:
Please align the parameters like this: https://github.com/eclipse/lemminx/blob/022615e0c9fb2cf7af12b1ad29a16380a2768a04/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/StringUtils.java#L434-L444 thank you